repository_service_tuf/helpers/tuf.py

```diff
@@ -30,23 +32,48 @@
 repository_metadata: Dict[str, Metadata] = {}
 
 
+class Roles(Enum):
+    ROOT = "root"
+    TARGETS = "targets"
+    SNAPSHOT = "snapshot"
+    TIMESTAMP = "timestamp"
+    BINS = "bins"
+
+
 @dataclass
-class RolesKeysInput:
-    expiration: int = 1
-    num_of_keys: int = 1
-    threshold: int = 1
-    keys: Dict[str, Any] = field(default_factory=dict)
-    offline_keys: bool = True
-    number_hash_prefixes: int = 0
-    paths: Optional[List[str]] = None
+class ServiceSettings:
+    number_of_delegated_bins: int = 256
+    targets_base_url: str = ""
+    targets_online_key: bool = True
 
     def to_dict(self):
         return asdict(self)
 
 
-def initialize_metadata(
-    settings: Dict[str, RolesKeysInput], save=True
-) -> Dict[str, Metadata]:
+@dataclass
+class RSTUFKey:
+    key: Optional[dict] = field(default_factory=dict)
+    key_path: Optional[str] = None
+    error: Optional[str] = None
+
+
+@dataclass
+class Setup:
```

Comment:
Do you think it's good to mention in the class documentation or the class name that this is a setup specifically for the `bootstrap ceremony`?
One way will be to rename it to `BootstrapSetup` and another option is to write a short comment.